### PR TITLE
add options for z-scores

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

File: gnomad_constraint/pipeline/constraint_pipeline.py [hunk 1]
```diff
@@ -901,14 +904,39 @@ def main(args):
...
     compute_constraint_args.add_argument(
-        "--raw-z-outlier-threshold",
-        help=(
-            "Value at which the raw z-score is considered an outlier. Values below the"
-            " negative of '--raw-z-outlier-threshold' will be considered outliers for"
-            " lof and missense varaint counts (indicating too many variants), whereas"
-            " values either above '--raw-z-outlier-threshold' or below the negative of"
-            " '--raw-z-outlier-threshold' will be considered outliers for synonymous"
-            " varaint counts (indicating too few or too many variants)."
+        "--raw-z-outlier-threshold-lof",
```
Comment (jkgoodrich) [inline gnomad_constraint/pipeline/constraint_pipeline.py:907]: Should we have these also have lower in the name just to be very obvious?

**Review by jkgoodrich — COMMENTED** [review]
this is much clearer than what we had before. It looks good, just one small suggestion for clarity

File: gnomad_constraint/pipeline/constraint_pipeline.py (hunk 1 above)
Comment (klaricch) [reply gnomad_constraint/pipeline/constraint_pipeline.py:907]: will add

**Review by jkgoodrich — APPROVED** [review]
LGTM!

**Pull request merged**